AdminUi/apps/admin_ui/lib/core/modals/confirmation_dialog.dart

```diff
@@ -33,7 +34,7 @@ class _ConfirmationDialog extends StatelessWidget {
       actions: [
         OutlinedButton(
           onPressed: () => Navigator.of(context).pop(false),
-          child: Text(context.l10n.cancel),
+          child: Text(cancelActionText),
```

Comment:
I'd like to make the cancelActionText optional and default to context.l10n.cancel:

```suggestion
          child: Text(cancelActionText ?? context.l10n.cancel),
```